libbeat/processors/add_process_metadata/docs/add_process_metadata.asciidoc

```diff
@@ -65,3 +69,21 @@ set to `true`, this condition will be ignored.
 `restricted_fields`:: (Optional) By default, the `process.env` field is not
 output, to avoid leaking sensitive data. If `restricted_fields` is `true`, the
 field will be present in the output.
+
+`host_path`:: (Optional) By default, the `host_path` field is set to the root
+directory of the host `/`. This is the path where `/proc` is mounted. For
+different runtime configurations of Kubernetes or Docker, the `host_path` can 
+be set to overwrite the default.
+
+`cgroup_prefixes`:: (Optional) By default, the `cgroup_prefixes` field is set
+to `/kubepods` and `/docker`. This is the prefix where the container ID is
+inside cgroup. For different runtime configurations of Kubernetes or Docker,
+the `cgroup_prefixes` can be set to overwrite the defaults.
+
+`cgroup_cache_expire_time`:: (Optional) By default, the 
+`cgroup_cache_expire_time` is set to 5 minuses. This is the length of time
```

Comment:
it is set to 30 seconds:
https://github.com/elastic/beats/pull/15947/files#diff-2ccbb70ee27fbd9dfb4a9c83b1c00acfR38

Reply:
Thanks, fixed it